polkadot/primitives/src/v6/mod.rs

```diff
@@ -1783,24 +1783,27 @@ impl<T: Encode> WellKnownKey<T> {
 
 /// Type discriminator for PVF preparation timeouts
 #[derive(Encode, Decode, TypeInfo, Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub enum PvfPrepTimeoutKind {
+pub enum PvfPrepKind {
 	/// For prechecking requests, the time period after which the preparation worker is considered
 	/// unresponsive and will be killed.
 	Precheck,
 
 	/// For execution and heads-up requests, the time period after which the preparation worker is
 	/// considered unresponsive and will be killed. More lenient than the timeout for prechecking
 	/// to prevent honest validators from timing out on valid PVFs.
```

Comment:
Old comments.

Reply:
This one, I don't get what has to be changed, we didn't change the behaviour of the PvfPrepKind.

Reply:
But we did change the name. ;) You're right that we only use this type for timeouts at the moment, but the name is more general than that. Is it reasonable to remove all the docs here about timeouts, and move them to `fn pvf_prep_timeout`?

Reply:
Oh, I see it does. I thought that centralizing everything in the enum would be a good idea, but it makes sense to leave it to the function. I've changed both.